### PR TITLE
Fix Image Uploading, Messaging UI, and add Prompts

### PR DESCRIPTION
**Pull request by ReadeP**
## Overview

Fixed image uploading, messaging ui/scroll issues, and added prompts to profile screens. Also fixed a lot of other UI bugs before launch.

## Changes Made

- Added Prompts to `ProfileViewController` and `MatchViewController`
- Fixed image uploading
- Fixed constraint, keyboard, and scrolling bugs in `ChatViewController`
- Fixed corner radius errors throughout the app
- Fixed profile interests getting cut off in the `CommunityViewController`
- Added support for Grad Students in the bio of `ProfileViewController`

## Test Coverage

Tested on device.

### CODE REVIEW

File: Pear/Controller/ChatViewController.swift [hunk 1]
```diff
@@ -203,32 +208,34 @@ class ChatViewController: UIViewController {
             self.databaseRef.child(pairMessagesPath).updateChildValues([messageId: 1])
         }
     }
+    
+    @objc func keyboardWillShow(notification: NSNotification) {
+        if let keyboardSize = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue {
+            if self.view.frame.origin.y == 0 {
+                self.view.frame.origin.y -= keyboardSize.height
```
Comment (Npikielny) [inline Pear/Controller/ChatViewController.swift:215]: I think `self` is unnecessary here–same with 221-222

File: Pear/Networking/NetworkManager.swift [hunk 2]
```diff
@@ -30,6 +30,8 @@ class NetworkManager {
             "bucket": "pear",
             "image": "data:image/png;base64,\(base64)"
         ]
+        
+
```
Comment (Npikielny) [inline Pear/Networking/NetworkManager.swift:34]: This seems excessive

File: Pear/View/PromptResponseView.swift [hunk 3]
```diff
@@ -1,32 +1,34 @@
 //
-//  ProfilePromptItemTableViewCell.swift
+//  PromptResponseView.swift
 //  Pear
 //
-//  Created by Amy Chin Siu Huang on 8/17/21.
+//  Created by Reade Plunkett on 9/23/21.
```
Comment (Npikielny) [inline Pear/View/PromptResponseView.swift:5]: Rude

File: Pear/Controller/NoMatchViewController.swift [hunk 4]
```diff
@@ -40,12 +40,20 @@ class NoMatchViewController: UIViewController {
...
-        noMatchTitleLabel.textColor = .black
-        noMatchTitleLabel.font = ._24CircularStdMedium
+        
+        let attributedTitle = NSMutableAttributedString()
+        
+        attributedTitle.append(NSAttributedString(string: "Meet your new ", attributes: [.font : UIFont._24CircularStdMedium, .foregroundColor : UIColor.black]))
+        
+        attributedTitle.append(NSAttributedString(string: "Pear", attributes: [.font : UIFont._24CircularStdMedium, .foregroundColor : UIColor.darkGreen]))
+        
+        attributedTitle.append(NSAttributedString(string: "\nnext Sunday", attributes: [.font : UIFont._24CircularStdMedium, .foregroundColor : UIColor.black]))
```
[the remark this answers is not in the document]
Comment (danielVebman) [reply Pear/Controller/NoMatchViewController.swift:54]: I think Noah means the new lines.

File: Pear/View/PromptResponseView.swift [hunk 5]
```diff
@@ -1,32 +1,34 @@
...
 //  Copyright © 2021 cuappdev. All rights reserved.
 //
 
 import UIKit
 
-class ProfilePromptItemTableViewCell: UITableViewCell {
-
+class PromptResponseView: UIView {
+
```
Comment (danielVebman) [inline Pear/View/PromptResponseView.swift:12]: Delete tab and on 43

**Pull request merged**